frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -390,6 +400,56 @@ function ReaderHighlight:addToMainMenu(menu_items)
             separator = i == #highlight_style,
         })
     end
+    if Screen:isColorScreen() then
+        table.insert(menu_items.highlight_options.sub_item_table, {
+            text_func = function()
+                local saved_color = self.view.highlight.saved_color or "yellow"
```

Comment:
(Haven't yet looked at the drawing code) but does the presence of "yellow" around here hint that on color devices, highlights will now be yellow by default?
People used to gray will 1) get yellow highlights, or keep getting them gray? and 2) be able to revert to our old same gray?
If yellow, are we fine with force-migrating people to a colorfull world ? Might be fine to advertize the feature - just asking to make us all aware of that.

Reply:
"yellow" is the current color default (when colors are enabled). If you disable color rendering then gray will be used.
I hadn't thought about this being a breaking change, but I suppose it is a change of defaults.
The "Apply default style to all highlights" button could be used to change all old highlights to a new color (including gray), so it is fairly easy to fix (although it is still a step nonetheless)

Reply:
Chosing "Gris" will give the same gray people used to have?

Reply:
I fixed the default gray color to match our legacy default grayscale HL ;).